core/src/services/azblob/core.rs

```diff
@@ -126,6 +128,10 @@ impl AzblobCore {
             req = req.header(http::header::RANGE, range.to_header());
         }
 
+        if let Some(etag) = if_none_match {
```

Comment:
Would you like to keep the variable name consistent? I was a little confused at first galance.

Reply:
Oops, I didn't know that I can do shadowing in Rust like this. Thanks for pointing out. I have already updated the pr.